server/data/lostBedClient.test.ts

```diff
@@ -68,4 +68,67 @@ describeClient('LostBedClient', provider => {
       expect(result).toEqual(lostBed)
     })
   })
+
+  describe('get', () => {
+    it('should get all lostBeds for a premises', async () => {
+      const lostBeds = lostBedFactory.buildList(2)
+
+      provider.addInteraction({
+        state: 'Server is healthy',
+        uponReceiving: 'A request to get lost beds',
+        withRequest: {
+          method: 'GET',
+          path: '/premises/premisesId/lost-beds',
```

Comment:
I would prefer if we could use the 'static-path' path from `server/paths/api.ts` here (same for line 117). I appreciate we might not be doing this everywhere in the client tests but I would like for it to be this way for new tests we write